pkg/code-generator/cmd/main.go

```diff
@@ -238,6 +238,9 @@ func (r *Runner) Run() error {
 
 	var customCompilePrefixes []string
 	for _, relativePath := range r.Opts.CustomCompileProtos {
+		if !strings.HasPrefix(relativePath, anyvendor.DefaultDepDir) {
```

Comment:
out of curiosity, what does this change do?

Reply:
There is an option in solo-kit to add custom protos to be compiled along by protoc. The compiler requires them to have the anyvendor prefix, but they are often not passed in with that prefix. So this code adds it just in case.